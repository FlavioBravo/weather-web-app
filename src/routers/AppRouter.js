import React from "react";
import {
  BrowserRouter as Router,
  Switch,
  Route
} from "react-router-dom";
import { Forecast } from "../components/forecast/Forecast";
import { Home } from "../components/home/Home";

export const AppRouter = () => {
  return (
    <Router>
      <div className="main">
        <Switch>
          <Route exact path="/" component={Home} />

          <Route exact path="/forecast/:cityId" component={Forecast} />
        </Switch>
      </div>
    </Router>
  );
}