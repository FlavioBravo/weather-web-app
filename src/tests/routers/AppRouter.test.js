import React from 'react';
import { shallow } from 'enzyme';
import { AppRouter } from '../../routers/AppRouter';

describe('Pruebas en <AppRouter />', () => {
    
    test('debe de mostrarse correctamente', () => {

        const wrapper = shallow( <AppRouter /> );
        expect( wrapper ).toMatchSnapshot();
        
    });

})