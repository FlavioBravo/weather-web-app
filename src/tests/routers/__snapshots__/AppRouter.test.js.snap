// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <AppRouter /> debe de mostrar el Home por defecto 1`] = `
<BrowserRouter>
  <div
    className="main"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/forecast/:cityId"
      />
    </Switch>
  </div>
</BrowserRouter>
`;

exports[`Pruebas en <AppRouter /> debe de mostrarse correctamente 1`] = `
<BrowserRouter>
  <div
    className="main"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/forecast/:cityId"
      />
    </Switch>
  </div>
</BrowserRouter>
`;
